main/midnight/CMakeLists.txt

```diff
@@ -196,7 +196,7 @@ if(ANDROID)
 #    list(APPEND GAME_SOURCE ${common_content_files})
 elseif(WINDOWS)
     if(NOT WINRT)
-        add_compile_options(-D_MSC_EXTENSIONS -D_OS_DESKTOP_ -D_OS_WIN32_)
```

Comment:
Fix for compilation, no longer allowed to define this, see: https://learn.microsoft.com/en-us/cpp/overview/cpp-conformance-improvements?view=msvc-170#cant-modify-implementation-reserved-macros